### ISSUE TITLE
If a resource is created by uv-l-relationalToCKAN, the download URL is not correct both in IC and PC

### ISSUE THREAD

**Issue by OskarStoffan** [OpenDataNode/open-data-node#232]
If a resource is created by uv-l-relationalToCkan DPU in the pipeline, the download URLs are:
not clickable in IC
not correct in IC
not clickable in PC
not correct in PC
'Download resource' button doesn't work in PC
'Data API' button is missing in PC
![ic](https://cloud.githubusercontent.com/assets/8984799/10411642/92cc531c-6f6c-11e5-955f-3b20b6ccbe66.png)
![pc](https://cloud.githubusercontent.com/assets/8984799/10411643/92cdf6b8-6f6c-11e5-86a9-424d7df9a04d.png)

**Comment by mvi-eea-sk** [comment]
The URL issue can be solved by adding parameter ``` qualified=True ``` to url_for function used here:
https://github.com/OpenDataNode/odn-ckan/blob/odn/release-v2.3/ckanext/datastore/plugin.py#L272-L274
This will cause, that the full URL will be displayed and will be dynamically constructed. Meaning if you access katalog with:
http://demo.ckan.org/dataset/...
The resource URL will be too http//demo.ckan.org/datastore/dump/....
If accessed through IP the resource URL will be too with IP
This will also apply to API calls showing resource.

@skrchnavy we discussed this issue few months ago and decided to not fix it (yet?).

The second issue with the "Download resource" button, I looked at the code and it was only displayed when the resource view failed to display too.
